core/templates/dev/head/components/rubrics-editor/rubrics-editor.directive.html

```diff
@@ -11,10 +11,14 @@
              title="Edit Rubric Explanation">&#xE254;
           </i>
         </div>
-        <span ng-if="$ctrl.getRubrics()[$index].getExplanation() === ''"
+        <span ng-if="$ctrl.getRubrics()[$index].getExplanation() === '<p></p>' && $index === 1"
```

Comment:
Why is one of these `<p></p>` and the other ''?

If the two are meant to mean the same thing (conceptually), then maybe defining an isExplanationEmpty() function and using that in both places might be better.

Reply:
Yeah, both are the same. Just that if the RTE becomes empty after it got values, it will be `<p></p>`, will write a function.

Reply:
Done